src/schemas/v2/combined.json

```diff
@@ -36814,6 +36814,182 @@
+                "type": "string",
+                "enum": [
+                  "Freehold",
+                  "Managed Freehold",
+                  "Commonhold",
+                  "Leasehold",
+                  "Other"
+                ]
+              },
+              "BasicDetails": {
+                "title": "Basic Property Details",
+                "type": "object",
+                "properties": {
+                  "bedrooms": {
+                    "title": "Number of bedrooms",
+                    "type": "integer"
+                  },
+                  "bathrooms": {
+                    "title": "Number of bathrooms",
+                    "type": "integer"
+                  },
+                  "yearBuilt": {
+                    "title": "Built Year",
+                    "type": "string",
+                    "format": "date"
+                  },
+                  "floorArea": {
+                    "title": "Floor area",
+                    "type": "object",
+                    "properties": {
+                      "inSquareMeter": {
+                        "type": "object",
+                        "properties": {
+                          "area": {
+                            "title": "Area in sqaure meters",
+                            "type": "number"
+                          },
+                          "pricePerSquareMeter": {
+                            "title": "Price per square meter",
+                            "type": "number"
+                          }
+                        }
+                      },
+                      "inSquareFoot": {
+                        "type": "object",
+                        "properties": {
+                          "area": {
+                            "title": "Area in sqaure foot",
+                            "type": "number"
+                          },
+                          "pricePerSquareFoot": {
+                            "title": "Price per square foot",
+                            "type": "number"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              "sourceUrls": {
+                "title": "Urls that point to property images, maps, etc.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "url": {
+                      "title": "URL",
+                      "type": "string",
+                      "format": "uri"
+                    }
+                  }
+                }
+              },
+              "propertyPricing": {
+                "title": "Property Pricing Estimates",
+                "description": "Approximate or calculated values assigned to a property to estimate its financial worth.",
+                "type": "object",
+                "properties": {
+                  "rentalEstimate": {
```

Comment:
is is per week, calendar month or year? We should either choose one (pcm?) or include a time period option

Reply:
added per week pcm, and year as rental frequency